src/components/Elements/Avatar/Avatar.tsx

```diff
@@ -63,13 +63,21 @@ const DisplayAvatar = ({
       />
     )
   } else if (avatar) {
-    content = <Avatar h={`${size}px`} w={`${size}px`} src={avatar} {...rest} />
+    content = (
+      <Avatar
+        h={`${size}px`}
+        w={`${size}px`}
```

Comment:
When `width` and `height` is the same, you can use boxSize
```suggestion
        boxSize={`${size}px`}
```